jenkins/scripts/coverage/generate-index-html.py

```diff
@@ -72,8 +79,13 @@
   <div id="main">
     <div class="container">
       <h1>Node.js Nightly Code Coverage</h1>
+      Nightly combined JavaScript coverage, across Windows, Linux, and a variety of runtimes,
+      can be found on <a href="https://codecov.io/gh/nodejs/node">codecov.io</a>:
+      <a href="https://codecov.io/gh/nodejs/node">
+        <img id="badge" alt="Combined coverage" src="https://codecov.io/gh/nodejs/node/branch/master/graph/badge.svg" />
+      </a>
       <h3>
-        Node.js Core&nbsp;&nbsp;<a href="https://github.com/nodejs/node">&rarr;</a>
+        Nightly Linux Coverage&nbsp;&nbsp;<a href="https://github.com/nodejs/node">&rarr;</a>
```

Comment:
Should this be more descriptive? The above text tells you to look elsewhere for "nightly ... linux" coverage but this is also "nightly linux coverage". Is this is just stuff we run ourselves? Should we describe the difference a bit more?

Reply:
Perhaps, "Nightly Linux Only".

As it sits today, the main difference is:

* we host it ourselves.
* it does not combine coverage from Windows, Linux, etc.
* it has block coverage, whereas the combined coverage is line coverage only (this is a bug with codecov which I believe is being worked on).

Reply:
> it has block coverage,

That should be branch coverage, and yeah, that one's a big deal for me. But I might be the only one who cares, to be honest! The combined Windows coverage is A+++++ and I'm looking forward to also get a non-Intl build merged in there as well, as that will also cover some otherwise-uncovered code. 🎉

Reply:
I'm OK with whatever you two think appropriate, just flagging that it _seems_ a bit, but I don't tend to consume these pages myself anyway.

Reply:
I think the branch coverage is useful for helping encourage new contributions, once we have this on `codecov.io`; and have seen it working for a few weeks, I'd advocate retiring the self-hosted builds...

I talked to codecov.io, and there's literally just an `if` statement that turns off block coverage for JavaScript. CC: @thomasrockhu.

Reply:
I went ahead and called the linux run, "Nightly Branch Coverage", since this is the main reason we want it to stick around for the time being.